### PR TITLE
Feat/cmc 1755 multiparty rpa workaround

### PR DESCRIPTION
**Pull request by jeejavr**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-1755

Change description
Setup the new email address for multiparty
Update to RoboticsEmailConfiguration to add new recipient for multiparty
Updates to RoboticsNotificationService to configure and send the email
Added new method to get the latest event trigger
Added second claimant and second defendant in RoboticsDataMapper
application yaml file update to add the new recipient
Added new test cases and updated accordingly)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapperTest.java [hunk 1]
```diff
@@ -180,6 +182,49 @@ void setup() {
...
+
+            assertThat(eventHistory).isNotNull();
+            assertThat(eventHistory)
+                .extracting("miscellaneous")
+                .asList()
+                .containsExactly(claimIssuedEvent, claimNotifiedEvent, claimDetailsNotifiedEvent);
+            String triggerReason = findLatestEventTriggerReason(eventHistory);
+            assertTrue(triggerReason.matches("Claim details notified."));
+        }
+
```
Comment (akriti-hmcts) [inline src/test/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapperTest.java:227]: this was not required for this story as mapper update is removed but it is ok to leave it as it is just  testing extra functionality

**Review by akriti-hmcts — DISMISSED** [review]

File: src/test/java/uk/gov/hmcts/reform/civil/service/robotics/RoboticsNotificationServiceTest.java [hunk 2]
```diff
@@ -106,7 +109,35 @@ void shouldSendNotificationEmail_whenCaseDataIsProvided() {
...
+
+        boolean multiPartyScenario = isMultiPartyScenario(caseData);
+        service.notifyRobotics(caseData, multiPartyScenario);
+
+        verify(sendGridClient).sendEmail(eq(emailConfiguration.getSender()), emailDataArgumentCaptor.capture());
+
+        EmailData capturedEmailData = emailDataArgumentCaptor.getValue();
+        String reference = caseData.getLegacyCaseReference();
+        String message = format(
+            "Mulitparty claim data JSON is attached for %s",
```
Comment (HarryH96) [inline src/test/java/uk/gov/hmcts/reform/civil/service/robotics/RoboticsNotificationServiceTest.java:134]: I dont think we are attaching the JSON payload for these emails anymore. Im not sure what the content should be of the message - doesnt mention anything specific on the ticket

File: src/main/java/uk/gov/hmcts/reform/civil/service/robotics/RoboticsNotificationService.java [hunk 3]
```diff
@@ -51,4 +55,42 @@ private EmailData prepareEmailData(CaseData caseData) {
...
         }
     }
+
+    private EmailData prepareEmailDataMultiParty(CaseData caseData) {
+        RoboticsCaseData roboticsCaseData = roboticsDataMapper.toRoboticsCaseData(caseData);
+        String triggerEvent = findLatestEventTriggerReason(roboticsCaseData.getEvents());
+
+        return EmailData.builder()
+            .message(String.format(
+                "Mulitparty claim data JSON is attached for %s",
```
Comment (HarryH96) [inline src/main/java/uk/gov/hmcts/reform/civil/service/robotics/RoboticsNotificationService.java:65]: I dont think we are attaching the JSON payload for these emails anymore. Im not sure what the content should be of the message - doesnt mention anything specific on the ticket

Comment (jeejavr) [reply src/main/java/uk/gov/hmcts/reform/civil/service/robotics/RoboticsNotificationService.java]: updated message body

**Review by HarryH96 — APPROVED** [review]

**Comment by HarryH96** [comment]
Nice work Jeeja :D

**Review by akriti-hmcts — APPROVED** [review]

**Review by karthick-mohanasundaram-HMCTS — APPROVED** [review]

**Review by akriti-hmcts — APPROVED** [review]

**Review by sherlynkhaw — DISMISSED** [review]

**Review by sherlynkhaw — APPROVED** [review]

**Review by karthick-mohanasundaram-HMCTS — APPROVED** [review]

**Comment by karthick-mohanasundaram-HMCTS** [comment]
@jeejavr could you please take a look at the failing checks? Thanks

**Pull request merged**